modules/ROOT/pages/import-and-export-preferences-studio.adoc

```diff
@@ -0,0 +1,37 @@
+= Export and Import Studio Preferences
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can export your Anypoint Studio preferences and import them into a new version.
+
+Follow this process if you had to download a fresh copy of Studio and you want to recover your previous configurations.
+
+== Export Your Studio Configurations
+
+. In Studio, go to *Preferences*.
+. Ensure that you applied all the configurations that you want to export.
+. Select the *Export* icon image:export-icon.png[30,30]
++
+image::export-preferences.png[]
+. Configure the preferences that you want to export:
++
+image::export-preferences-configure.png[]
+<1> Select *Export all* to export all of your preferences, or select the specific configurations that you want to export.
+<1> Define a location and name for your exported preferences.
```

Comment:
Actually, what about replacing Step 4 with Steps 4 and 5:
4. Select *Export all*...or select....
5. Define a location and name....

"Define"? Should that be "Supply"?

Reply:
...and then WHERE is this location and what type of name? Does the user know the location/name of the fresh Studio instance? Is this configuration send to the new instance...and if so, why does the new instance have to re-select them?
If I am asking newbie questions that your audience will not, please forgive the churn. I know you will do what is best.